system/ee/ExpressionEngine/Addons/member/upd.member.php

```diff
@@ -80,6 +83,11 @@ public function update($current = '')
             ee()->db->update('actions', ['method' => 'do_member_search']);
         }
 
+        if (version_compare($current, '6.3.4', '<')) {
```

Comment:
In case of add-ons updater, `$current` represents the add-on version, not EE version. So you would need to update version in `addon.setup.php` and use corresponding version in this check

Reply:
oh, good to know; updated and pushed anew :)